### DISCUSSION TITLE
Version 9.9.0.3

### DISCUSSION THREAD

**Discussion by FilipePS** [FilipePS/Traduzir-paginas-web#712] [Announcements]
- Update interface translations

<hr /><em>This discussion was created from the release <a href='https://github.com/FilipePS/Traduzir-paginas-web/releases/tag/v9.9.0.3'>Version 9.9.0.3</a>.</em>

**Comment by DongXiangWings** [comment]
Excuse me. I can't use Google translate on pages & words translate in the recent versions. It renders  blank on the translation page, but works fine using Bing and Yandex.

**Comment by Lemon514293676** [comment]
微软翻译基本上完善了，看来可以加入腾讯翻译了
